newIDE/app/src/EventsSheet/ParameterFields/ForceMultiplierField.js

```diff
@@ -102,8 +101,8 @@ export default class ForceMultiplierField extends Component<
 export const renderInlineForceMultiplier = ({
   value,
 }: ParameterInlineRendererProps) => {
-  if (value === '1') return 'a permanent';
-  else if (value === '0' || value === '') return 'an instant';
+  if (value === '1') return <Trans>{`a permanent`}</Trans>;
+  else if (value === '0' || value === '') return <Trans>{`an instant`}</Trans>;
 
-  return 'a (multiplier: ' + value + ')';
+  return `a (multiplier: ' + ${value} + ')`;
```

Comment:
This is not correct ;)

Reply:
The original was  

https://github.com/4ian/GDevelop/blob/bb296dc34e4af818ac45f48450b1f6691b9d0cc9/newIDE/app/src/EventsSheet/ParameterFields/ForceMultiplierField.js#L108

Reply:
Sorry for the silly mistake. 
Template string now converted back to original string.